.github/workflows/run.yml

```diff
@@ -45,13 +45,13 @@ jobs:
         run: bin/dfx-sns-demo
       - name: Create and finalize an additional SNS
         run: |
-          ./bin/dfx-sns-demo-mksns
+          ./bin/dfx-sns-demo-mksns -i 1
           ./bin/dfx-sns-sale-buy
           ./bin/dfx-sns-sale-finalize
       - name: Create lots of SNS
         if: matrix.os != 'macos-12'
         run: |
-          ./bin/dfx-sns-demo-mksns-parallel -s 10 -m snsdemo8
+          ./bin/dfx-sns-demo-mksns-parallel -s 10 -o 2 -m snsdemo8
```

Comment:
Can we use the long name?
(I know the others aren't long either. Feel free to change them as well or keep them unchanged.)